config/secrets.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  secret_key_base: 8b16915e35590d84c65509821a16dd29561866c937e46d0cce6be9e815b60a0cea81ad96ed5800d56a5de9b31f9bf0a32a45d7435e7ffd91aa52bb185ce64555
+
+test:
+  secret_key_base: 53717d6a1d28dc695039f211666e221263891e60e5a19259c22d025f4a601c8ac3b3ed16ec5878863731adbe1dc6f7f1f4b5b4d006b79e19847d5ca44a2eb708
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
+production:
```

Comment:
For the Ruby services, though they have been upgraded to Rails 4.2, we *don't* need `secrets.yml`, and I think it's because session storage is disabled:
https://github.com/change/petition_service/blob/a536d4793b8d62113fd1bfd36020268148f61749/config/application.rb#L96

For monorail we probably can't disable session storage, but for the gem perhaps we can...